docs/Reference/ContentTypeParser.md

```diff
@@ -87,6 +87,28 @@ fastify.addContentTypeParser('application/vnd.custom', (request, body, done) =>
 fastify.addContentTypeParser('application/vnd.custom+xml', (request, body, done) => {} )
 ```
 
+### Important: Using addContentTypeParser with fastify.register
```

Comment:
```suggestion
### Using addContentTypeParser with fastify.register
```

One last suggestion, It being a heading should draw attention anyway, so declaring it's important is not needed.